internal/webhooks/machineset.go

```diff
@@ -136,6 +136,10 @@ func (webhook *MachineSet) Default(ctx context.Context, obj runtime.Object) erro
 		m.Spec.Template.Spec.InfrastructureRef.Namespace = m.Namespace
 	}
 
+	if m.Spec.Template.Spec.MinReadySeconds == nil {
```

Comment:
Q: can we drop this now that we have // +default=0 on the field definition?

Reply:
We need to keep it because the +default=0 isn't used.